CMakeLists.txt

```diff
@@ -550,12 +550,12 @@ if (WIN32)
     "${DDK_ROOT}/bin/${ARCH_SFX}/dbghelp.dll"
     "${DDK_ROOT}/tools/tracing/${DDK_SFX}/dbghelp.dll"
     "${PROGFILES}/Microsoft Visual Studio */Common7/IDE/Remote Debugger/${ARCH_SFX}/dbghelp.dll"
-    "${PROGFILES32}/Windows Kits/*/Debuggers/${ARCH_SFX}/dbghelp.dll")
+    "${PROGFILES32}/Windows Kits/*/Debuggers/${ARCH_SFX}/dbghelp.dll"
+    # For Visual Studio 2010+, x64 dbghelp.dll resides in Program
+    # Files (x86) rather than Program Files.
+    "${PROGFILES32}/Microsoft Visual Studio */Common7/IDE/Remote Debugger/${ARCH_SFX}/dbghelp.dll")
```

Comment:
I don't understand: what is the bug?  And how does this fix it?  The comment says it applies to x64 only, so why does moving it out here change anything?  Could you add an explanation of what the problem is to the issue tracker?